creator-node/src/tracer.ts

```diff
@@ -0,0 +1,153 @@
+import type {
+  Span,
+  SpanOptions,
+  SpanContext,
+  AttributeValue
+} from '@opentelemetry/api'
+import {
+  trace,
+  diag,
+  DiagConsoleLogger,
+  DiagLogLevel,
```

Comment:
looks like not all of these deps are used?

Reply:
removing